PowerEditor/src/langs.model.xml

```diff
@@ -72,7 +72,7 @@
             <Keywords name="instre1">7z adduser alias apt-get ar as asa autoconf automake awk banner base64 basename bash bc bdiff blkid break bsdcpio bsdtar bunzip2 bzcmp bzdiff bzegrep bzfgrep bzgrep bzip2 bzip2recover bzless bzmore c++ cal calendar case cat cc cd cfdisk chattr chgrp chmod chown chroot chvt cksum clang clang++ clear cmp col column comm compgen compress continue convert cp cpio crontab crypt csplit ctags curl cut date dc dd deallocvt declare deluser depmod deroff df dialog diff diff3 dig dircmp dirname disown dmesg do done dpkg dpkg-deb du echo ed egrep elif else env esac eval ex exec exit expand export expr fakeroot false fc fdisk ffmpeg fgrep fi file find flex flock fmt fold for fsck function functions fuser fusermount g++ gas gawk gcc gdb genisoimage getconf getopt getopts git gpg gpgsplit gpgv grep gres groff groups gunzip gzexe hash hd head help hexdump hg history httpd iconv id if ifconfig ifdown ifquery ifup in insmod integer inxi ip ip6tables ip6tables-save ip6tables-restore iptables iptables-save iptables-restore ip jobs join kill killall killall5 lc ld ldd let lex line ln local logname look ls lsattr lsb_release lsblk lscpu lshw lslocks lsmod lsusb lzcmp lzegrep lzfgrep lzgrep lzless lzma lzmainfo lzmore m4 mail mailx make man mkdir mkfifo mkswap mktemp modinfo modprobe mogrify more mount msgfmt mt mv nameif nasm nc ndisasm netcat newgrp nl nm nohup ntps objdump od openssl p7zip pacman passwd paste patch pathchk pax pcat pcregrep pcretest perl pg ping ping6 pivot_root poweroff pr print printf ps pwd python python2 python3 ranlib read readlink readonly reboot reset return rev rm rmdir rmmod rpm rsync sed select service set sh sha1sum sha224sum sha256sum sha3sum sha512sum shift shred shuf shutdown size sleep sort spell split start stop strings strip stty su sudo sum suspend switch_root sync systemctl tac tail tar tee test then time times touch tr trap troff true tsort tty type typeset ulimit umask umount unalias uname uncompress unexpand uniq unlink unlzma unset until unzip unzipsfx useradd userdel uudecode uuencode vi vim wait wc wget whence which while who wpaste wstart xargs xdotool xxd xz xzcat xzcmp xzdiff xzfgrep xzgrep xzless xzmore yes yum zcat zcmp zdiff zegrep zfgrep zforce zgrep zless zmore znew zsh</Keywords>
         </Language>
         <Language name="batch" ext="bat cmd nt" commentLine="REM">
-            <Keywords name="instre1">assoc aux break call cd chcp chdir choice cls cmdextversion color com com1 com2 com3 com4 con copy country ctty date defined del dir do dpath echo else endlocal erase errorlevel exist exit for ftype goto if in loadfix loadhigh lpt lpt1 lpt2 lpt3 lpt4 md mkdir move not nul path pause popd prn prompt pushd rd rem ren rename rmdir set setlocal shift start time title type ver verify vol</Keywords>

```

Comment:
The `choice` command is a part of batch programming:
https://en.wikipedia.org/wiki/Choice_(command)

Reply:
From LexBatch.cxx:
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/c6e433f70ba2a8d80a3932bc76118d7c2e9707da/lexilla/lexers/LexBatch.cxx#L640-L644

The keywords in the `intrel1` tag is described as `"Internal Commands"`. `choice` is not an ***internal command***!

The wikipedia link describes `choice` as:

> The command was first introduced as an ***external command*** (with filenames CHOICE.COM or CHOICE.EXE) 

which is an ***external command*** printed in their own words! I disagree with you and even the wikipedia link disagrees with you.

Reply:
@mpheath 
Yes, you're right.
In this case, I think we should also colorize the external commands.
I'll see in the #14715.
Thank you for the reviewing!

Reply:
@donho
> In this case, I think we should also colorize the external commands with another category.

**The batch lexer is already styling *external* commands now!**

This image that I posted in a previous [comment](https://github.com/notepad-plus-plus/notepad-plus-plus/pull/14554#issuecomment-1879723688) containing some commands as an example:

![](https://private-user-images.githubusercontent.com/58158242/294696875-efa5e0e9-0929-49c0-88f8-9317250e01f2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc4OTE0NzIsIm5iZiI6MTcwNzg5MTE3MiwicGF0aCI6Ii81ODE1ODI0Mi8yOTQ2OTY4NzUtZWZhNWUwZTktMDkyOS00OWMwLTg4ZjgtOTMxNzI1MGUwMWYyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE0VDA2MTI1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAwODFkMGQxOTI0MTAwNGQzODYwNmVhYmNhNjUxMmQ0MDRjOGFjZTlhMDhhOTQyYWE5MTVjYTYwOTJjNzkyNzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.lcyzh8XgAs0ATP94DLanuE_gNofIK72ujTcNFhE_OZk)

The *internal* commands are *dark blue* and the *external* commands are *light blue* in the default theme. `choice` will be *light blue* instead of *dark blue* to let the user know that it is an *external* command.

If an *external* command list is defined, then **only** the commands in the list will be styled and all other will be default style.

See this code from `LexBatch.cxx`:

https://github.com/notepad-plus-plus/notepad-plus-plus/blob/978118dc0e3d805a2966167f06e572d9020f455f/lexilla/lexers/LexBatch.cxx#L376-L383

If not `keywords2` list, style as `SCE_BAT_COMMAND` (*Line 378*).
Else if `keywords2` list, then only commands in the list are styled (*Line 380*).
Else unrecognized as default style (*Line 382*).

If `keywords2` is defined with a list, **automatic styling** of *external* commands will no longer occur. I do not recommend creating an *external* command list else may need to define all possible *external* commands in the list and that would be a very large list. Instead, let the batch lexer handle what an external command is and what is not.

To show how `choice` looks currently in the XML *instre1* list or removed from the XML *instre1* list:
![choice](https://github.com/notepad-plus-plus/notepad-plus-plus/assets/58158242/566cf560-e31a-4cb9-a7c5-16e1be1a9fad)

The image is 2 gif frames which change about every 3 seconds.

Example batch code copied from https://www.computerhope.com/issues/ch001674.htm

Nothing special done except removed `choice` from the list. It becomes light blue and looks OK IMO. I do not see any problem with the change of removing `choice` from the *internal* command list. My advice, commit #14715 and move on to the next task.